packages/server/src/routes/aic/index.ts

```diff
@@ -0,0 +1,125 @@
+import express from 'express'
+import tid from './tid'
+import { TID } from '@adxp/common'
+import { update_tick } from '@adxp/aic'
+import * as crypto from '@adxp/crypto'
+import { z } from 'zod'
+
+// Note: do not use the dev/test key in production
+//       pull the prod key from the secret store
+export const CONSORTIUM_KEYPAIR = crypto.EcdsaKeypair.import(
```

Comment:
hey I'm pretty sure we want the AIC server outside of the PDS. Most folks that run a PDS do not want to run AIC. And we'll probably even run them as separate infrastructure.

also in terms of startup from the cli: I think we'll want that in the `dev-env` package but not the `cli` package